db/schema.rb

```diff
@@ -70,7 +70,6 @@
     t.datetime "updated_at",        null: false
     t.index ["account_id", "target_account_id"], name: "index_blocks_on_account_id_and_target_account_id", unique: true, using: :btree
   end
-  execute   "ALTER TABLE blocks ADD CONSTRAINT check_blocks_on_block CHECK(block = TRUE), INHERIT block_mutes"
```

Comment:
@akihikodaki Well, did you added this line manually in your PR? If so, it will be removed all future db changes...